02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -1,28 +1,37 @@
+        - [The "Frequency" of a Change](#the-frequency-of-a-change)
+        - [How to Make Maintenance Easier](#how-to-make-maintenance-easier)
+    - [Criteria for "Good" Documentation](#criteria-for-good-documentation)
+    - [Evaluating PureScript's Documentation](#evaluating-purescripts-documentation)
+- [Why is PureScript's Documentation Lacking and How Do We Improve It?](#why-is-purescripts-documentation-lacking-and-how-do-we-improve-it)
+- [New Learners: What is the Best Way to Learn PureScript?](#new-learners-what-is-the-best-way-to-learn-purescript)
+- [PureScript Documentation Writers: What is the Best Way to Write Documentation in this Context?](#purescript-documentation-writers-what-is-the-best-way-to-write-documentation-in-this-context)
+- [Core Contributors: ???](#core-contributors-)
+
+
 ## Why Read This Document?
 
-This document is not meant to convince you to use PureScript. We assume that readers already believe in using PureScript instead of alternatives.
+This document is not meant to convince you to use PureScript. We assume that readers already want to learn to use PureScript.
 
-It is well-known among the PureScript community that its documentation is lacking in some critical areas. As a result, @chexxor started the [PureScript documentation efforts in 2019](https://discourse.purescript.org/t/purescript-documentation-efforts-in-2019/524) discourse thread to answer one question:
+It is well-known amongst the PureScript community that its documentation is lacking in some critical areas. As a result, @chexxor started the [PureScript documentation efforts in 2019](https://discourse.purescript.org/t/purescript-documentation-efforts-in-2019/524) Discourse thread to answer one question:
 > How can we improve the PureScript documentation in 2019?
 
-(Purpose) The following document tries to do two things:
-1. Provide an answer to @chexxor's above question
-2. Answer other related questions that various audiences will have:
-    - new learners (regardless of language background)
+(Purpose) The following document tries to do the following things:
+1. Determine why PureScript's documentation is seen as lacking.
+2. Understand the expectations & needs of the different documentation consumers.
+    - New learners (regardless of language background)
     - PureScript documentation writers
-    - Core Contributors
-
-Outline
-- Explain what "good" documentation is and define a criteria
-- Evaluate PureScript's documentation using that criteria
-- Explain why PureScript's documentation is seen as lacking
-- Suggest plans to improve the current documentation situation
-- Address various audience's possible concerns or questions centered on these themes:
-    - New learners - How should I learn PureScript?
-    - PureScript documentation writers - How should I write good maintainable documentation for others?
-    - Core Contributors -
+    - Core contributors
+3. Explore strategies for improving PureScript's documentation.
```

Comment:
I think this is fine (meaning, fine to include some of our ideas for how things could improve). It seems to be the direction we keep naturally heading towards.